src/coreclr/inc/regdisp.h

```diff
@@ -155,7 +155,7 @@ inline TADDR GetRegdisplayStackMark(REGDISPLAY *display) {
 
 #elif defined(TARGET_64BIT)
 
-#if defined(TARGET_ARM64)
+#if defined(TARGET_ARM64) || defined(__ARM_HOST)
```

Comment:
Where is `__ARM_HOST` defined?

Reply:
This one I take from this answer https://stackoverflow.com/questions/23934862/what-predefined-macro-can-i-use-to-detect-the-target-architecture-in-clang I suspect that I should use `TARGET_XXX` since this is specific to runtime repo. Do not have time to properly look for the appropriate target.

Reply:
I think I should use `TARGET_ARM` here